### PR TITLE
Documenting that guild.chunk returns a list of members

### PR DESCRIPTION
**Pull request by Scrumpyy**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by vcokltfre — CHANGES_REQUESTED** [review]
While we're here, the return type of the function should be changed to a list of Members too `-> None` -> `-> List[Member]` (make sure List is imported from typing)

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
This can also be None?

File: nextcord/guild.py [hunk 1]
```diff
@@ -2845,7 +2845,7 @@ async def edit_widget(self, *, enabled: bool = MISSING, channel: Optional[Snowfl
 
         await self._state.http.edit_widget(self.id, payload=payload)
 
-    async def chunk(self, *, cache: bool = True) -> None:
+    async def chunk(self, *, cache: bool = True) -> List[Member]:
```
Comment (ooliver1) [inline nextcord/guild.py:2848]: This also needs to have the return type with optional

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: nextcord/guild.py [hunk 2]
```diff
@@ -2845,7 +2845,7 @@ async def edit_widget(self, *, enabled: bool = MISSING, channel: Optional[Snowfl
 
         await self._state.http.edit_widget(self.id, payload=payload)
 
-    async def chunk(self, *, cache: bool = True) -> None:
+    async def chunk(self, *, cache: bool = True) -> Optional(List[Member]):
```
Comment (ooliver1) [inline nextcord/guild.py:2848]: Isn't it Optional[] not Optional()

**Review by ooliver1 — CHANGES_REQUESTED** [review]

**Review by ooliver1 — APPROVED** [review]

**Review by vcokltfre — APPROVED** [review]

**Pull request merged**